testprojects/pants-plugins/src/python/python_constant/target_types_test.py

```diff
@@ -3,6 +3,13 @@
 from textwrap import dedent
 
 import pytest
+from python_constant import target_types
```

Comment:
Did your IDE do this automatically? I assume this will fail our lint checks now, since it didn't before...